crates/biome_service/src/workspace/scanner.rs

```diff
@@ -142,14 +136,6 @@ fn scan_folder(folder: &Utf8Path, ctx: ScanContext) -> Duration {
         }
     }));
 
-    ctx.workspace
-        .update_module_graph(WatcherSignalKind::AddedOrChanged, &handleable_paths);
-
-    let _ = ctx
-        .workspace
-        .notification_tx
-        .send(ServiceDataNotification::Updated);
```

Comment:
This is already done in `WorkspaceServer::open_file_by_scanner`, so it won't be needed here.